### PR TITLE
Setup basic components and structure

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
## What was done in this PR

- Setup basic components and structure.
- Files organized in Ducks structure.

### CODE REVIEW

File: src/components/books.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function Books() {
+  return (
+        <h1></h1>
```
Comment (vichuge) [inline src/components/books.js]: - [x] It'd be great if books.js show a list of books, please work on it. 🐶 
- [x] Please, create a book component for each book, these books could be listed on books.js according to the project requirements.

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Categories from './categories.js';
+import Books from './books.js';
```
Comment (vichuge) [inline src/components/App.js]: - [x] I couldn't observe a form to add new books, please work on that (don't forget the submit button). The form shouldn't work right now, it needs just to be there. 🌵

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/list.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    id: 2,
+  }];
+
+  return (
+        <ul>
+            {bookList.map((book) => (
+               <Books name={book.title} author={book.author} genre={book.genre} key={book.id}/>
+            ))}
+        </ul>
+  );
```
Comment (Davidosky007) [inline src/components/books/list.js]: * Like the previous reviewer pointed out you should have a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @GuilhermeGiachellin 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
cr2 gif_almost-there

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._

Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/list.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            ))}
+        </ul>
+        <form>
+              <h1 className="form_title">Add Books</h1>
+              <div className="input_ctn">
+              <input placeholder="Book Title" className="book_input"></input>
+              <input placeholder="Category" className="category_input"></input>
+              <button type="submit">ADD BOOK</button>
+              </div>
+        </form>
```
Comment (mikenath223) [inline src/components/books/list.js]: - [x] This should be a component as stated in the previous review. Please wrap this form as a new component.

Comment (GuilhermeGiachellin) [reply src/components/books/list.js]: Everything is working fine now? Could you check it for me? Thank you!

**Pull request closed without merging**